tests/integration/init/schemas/schemas_test_data_setup.py

```diff
@@ -43,28 +43,6 @@ def setUpClass(cls):
         setup_partner_schema_data("partner-registry", schemas_client)
         setup_schema_data_for_pagination("test-pagination", schemas_client)
         setup_non_partner_schema_data("other-schema", schemas_client)
-        # WHEN the user follows interactive init prompts
-        # 1: AWS Quick Start Templates
-        # 1: Hello World Example
-        # N: do not use DEFAULT hello world template
-        # 11: Java runtime
-        # 2: dependency manager maven
-        # eb-app-maven: response to name
-        # Y: clone/update the source repo
-        # 1: hello world
-
-        user_input = """
-1
-1
-N
-5
-1
-2
-eb-app-maven
-    """
-        with tempfile.TemporaryDirectory() as temp:
-            runner = CliRunner()
-            runner.invoke(init_cmd, ["--output-dir", temp], input=user_input)
```

Comment:
Why we removed this block, is this replaced with something else?

Reply:
Not sure what the use of running an init before the test class was or why this was introduced at all. This init flow is also very out-of-date and the tests pass without it.

Reply:
We also don't run any assertions on the output of this command and the inputs were wrong for a while.

Reply:
Yeap this looks weird to me as well. All setup stuff are done in the functions above, so this part shouldn't be required. Let's wait and see the canary runs

Reply:
These have all passed on Windows and Linux now